cmd/prometheus/uname_linux.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
Does the syscall really only work on linux? Otherwise the file can be renamed to `uname_unix.go`